jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Jackson1Annotator.java

```diff
@@ -44,8 +44,19 @@
  */
 public class Jackson1Annotator extends AbstractAnnotator {
 
+    private JsonSerialize.Inclusion inclusionLevel = JsonSerialize.Inclusion.NON_NULL;
+
     public Jackson1Annotator(GenerationConfig generationConfig) {
         super(generationConfig);
+        switch (generationConfig.getInclusionLevel()) {
```

Comment:
Could you format this switch statement the same as the one in Jackson2Annotator?